drivers/console/Kconfig

```diff
@@ -323,7 +323,7 @@ config XTENSA_CONSOLE_INIT_PRIORITY
 
 config NATIVE_POSIX_CONSOLE_INIT_PRIORITY
 	int "Init priority"
-	default 60
+	default 99
```

Comment:
What does this change have to do with building drivers as a static library?

Reply:
This is due to linking order. For example, if both native POSIX console and UART console are enabled, the UART console driver takes over the printk hook which means there is nothing being printed to the terminal. Tests are failing due to empty outputs.